sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceAsyncClient.java

```diff
@@ -128,10 +179,10 @@ public WebPubSubAuthenticationToken getAuthenticationToken(GetAuthenticationToke
      * Close the client connection.
      * @param connectionId Target connection Id.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @return the completion.
```

Comment:
ugh

Reply:
Need better javadoc in LLC autorest codegen. Filed an issue for autorest - https://github.com/Azure/autorest.java/issues/1178